corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -544,6 +544,9 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", function () {
                     self.selectValuesByKeys,
                     self.options.sidebarEnabled
                 );
+                if (self.smallScreenEnabled && self.options.sidebarEnabled) {
```

Comment:
This check maybe isn't strictly necessary, because jquery handles its own errors (as in the non-collapsible large screen sidebar or non-SSCS query screen with no sidebar), but it prevents jquery doing unnecessary work.